development/client/src/index.js

```diff
@@ -1,12 +0,0 @@
-import React from "react";
```

Comment:
Regarding this split into two projects - do you think it's possible that we have one page with both versions side by side?